syft/execution/worker_actions.py

```diff
@@ -0,0 +1,115 @@
+from syft.execution.action import Action
+from syft.workers.abstract import AbstractWorker
+
+from syft_proto.execution.v1.worker_action_pb2 import (
+    WorkerAction as WorkerActionPB,
+    # HACK replace this with the correct syft proto objects and update proto version in reqs
+    # and update documentations
+)
+
+WORKER_ACTION_METHODS = ["store", "load"]
+
+
+class WorkerAction(Action):
```

Comment:
I'm bad at naming stuff but I feel like this is a bit too generic, right? I mean, all actions are performed by a worker

Reply:
Would worker_storage_actions or worker_protocol_actions or role_actions or worker_role_actions be better?
I just used WorkerAction to get going because that's what Karl called it initially. Open for suggestions though.

Reply:
Yeah, bad naming on my part. Accurate, but not specific. 😓 

Based on your suggestions, what about `StorageActions`?